build/optaplanner-build-parent/pom.xml

```diff
@@ -61,7 +61,7 @@
     <!-- The first version that supports analysis of classes compiled by Java 11. -->
     <version.dependency.plugin>3.1.2</version.dependency.plugin>
     <version.surefire.plugin>2.22.2</version.surefire.plugin>
-    <version.versions.plugin>2.8.1</version.versions.plugin>
+    <version.versions.plugin>2.5</version.versions.plugin>
```

Comment:
Please add a comment with a link to the issue, so that we don't upgrade the version before it's resolved.

Reply:
@rsynek added